src/components/ItemDetail/ItemDetailCarousels.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import CarouselSection from '../CarouselSection';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const ItemDetailCarousels = props => {
+  const { adminSetItems, collectionItems, item } = props;
+  const styles = {
+    caret: {
+      fontSize: '6rem',
+      color: '#f0f0f0'
+    }
+  };
+
+  return (
+    <section className="contain-1120 item-section item-categories-wrapper">
+      <h3>Library Division and Collections with this Item:</h3>
+
+      <div className="expander expander1" data-collapse="data-collapse">
+        {item && collectionItems && <h3 className="open">Collection</h3>}
+        {item &&
+          collectionItems && (
+            <CarouselSection
+              sectionTitle={item.collection[0].title[0]}
```

Comment:
I'm getting an error here if I click on an item that is not in a Collection. 

```
TypeError: Cannot read property 'title' of undefined
ItemDetailCarousels
src/components/ItemDetail/ItemDetailCarousels.js:24
```

Reply:
Ok, I'll see what's up with that.

Reply:
@davidschober So, for now at least, if an item does NOT have a Collection, then the Collections carousel will not display on the Item Detail page.